webrtc.html

```diff
@@ -4757,27 +4757,7 @@ <h2>Methods</h2>
               <var>keygenAlgorithm</var> argument uses the WebCrypto
               [[!WebCryptoAPI]] <a data-cite=
               "!WebCryptoAPI#dfn-AlgorithmIdentifier">
-              <code>AlgorithmIdentifier</code></a> type. The
-              <var>keygenAlgorithm</var> value MUST be a valid argument to
-              <a data-cite=
-              "!WebCryptoAPI#SubtleCrypto-method-generateKey">
-              <code>window.crypto.subtle.generateKey</code></a>; that is, the
-              value MUST produce a non-error result when normalized according
-              to the WebCrypto <a data-cite=
```

Comment:
This requirement is lost. Is there equivalent language in the new PR to limit valid values here?

Reply:
This should be covered by step 5